Essentials/src/messages.properties

```diff
@@ -494,6 +494,7 @@ vanished=\u00a76You are now completely invisible to normal users, and hidden fro
 versionMismatch=\u00a74Version mismatch\! Please update {0} to the same version.
 versionMismatchAll=\u00a74Version mismatch\! Please update all Essentials jars to the same version.
 voiceSilenced=\u00a76Your voice has been silenced\!
+muteFormat=\u00a74 Reason: {0}
```

Comment:
Having spaces before the string here means that the string can only be used when concatenating it with another string. In addition, calling it `muteFormat` doesn't seem very consistent with the other uses of `-Format` - `muteReason` would be a more descriptive name for the string.